com.unity.render-pipelines.universal/Runtime/ForwardRenderer.cs

```diff
@@ -78,7 +78,7 @@ public ForwardRenderer(ForwardRendererData data) : base(data)
             m_MainLightShadowCasterPass = new MainLightShadowCasterPass(RenderPassEvent.BeforeRenderingShadows);
             m_AdditionalLightsShadowCasterPass = new AdditionalLightsShadowCasterPass(RenderPassEvent.BeforeRenderingShadows);
 #if ENABLE_VR && ENABLE_XR_MODULE
-            m_XROcclusionMeshPass = new XROcclusionMeshPass(RenderPassEvent.BeforeRenderingPrepasses);
```

Comment:
How does this change affect multipass? In multipass passes in BeforeRenderingPrepasses are executed once while passes executed in BeforeRenderingOpaques are executed twice.

Reply:
Essentially, in multipass, we need to render the occlusion mesh twice just before the opaques so we're good.
Note that occlusion mesh will not be rendered to the depth texture (prepass) because it's not giving us any perf back : in z-only pass we don't have heavy frag processing.